users/service.go

```diff
@@ -193,10 +194,11 @@ func (svc service) ListClients(ctx context.Context, token string, pm mgclients.P
 		Offset:   pm.Offset,
 		Limit:    pm.Limit,
 		Name:     pm.Name,
+		Id:       pm.Id,
 		Identity: pm.Identity,
 		Role:     mgclients.UserRole,
 	}
-	pg, err := svc.clients.SearchBasicInfo(ctx, p)
+	pg, err := svc.clients.SearchClients(ctx, p)
```

Comment:
Why do we need to pulicate the mgclients.Page, instead we can pass variable pm directly to SearchClients
```suggestion
    pg, err := svc.clients.SearchClients(ctx, pm)
```